components/movementsensor/gpsrtk/rtk_station.go

```diff
@@ -46,26 +42,28 @@ const (
 	timeMode  = "time"
 )
 
+var stationModel = resource.DefaultModelFamily.WithModel("rtk-station")
+
 // ErrStationValidation contains the model substring for the available correction source types.
-var ErrStationValidation = fmt.Errorf("only serial, I2C, and ntrip are supported correction sources for %s", stationModel.Name)
+var (
+	ErrStationValidation = fmt.Errorf("only serial, I2C are supported for %s", stationModel.Name)
+	errRequiredAccuracy  = errors.New("required Accuracy can be 1-5")
```

Comment:
required Accuracy can be 1-5 <- adding unit here would be nice.

Reply:
I don't think there is a unit it's just an arbitrary ranking, but I can add in that 5 is the highest accuracy.

Reply:
```suggestion
	errRequiredAccuracy  = errors.New("required Accuracy can be a fixed number in the range of 1-5, 5 being the highest accuracy")
```

Reply:
I don't actually think this is true.

Reply:
I vaguely remember a fix of 4 being best, but I can't find a resource on this. Happy to keep the error as is until I find the resource confirming this